policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/ns_basic_personal_amount/applicable_rate.yaml

```diff
@@ -0,0 +1,11 @@
+description: Nova Scotia applicable rate for calculating partial basic personal amount in basic personal amount.
```

Comment:
Change the description to be:
Nova Scotia employs this applicable rate for calculating partial basic personal amounts.